js/webviewGestures.js

```diff
@@ -128,10 +130,14 @@ window.addEventListener('wheel', function (e) {
 
     if (horizontalMouseMove < -150 && Math.abs(horizontalMouseMove / verticalMouseMove) > 2.5 && !hasShownSwipeArrow) {
       hasShownSwipeArrow = true
```

Comment:
```hasShownSwipeArrow``` would need to be moved inside the if(canGoBack) statement, both here and in the section below.